_topic_maps/_topic_map.yml

```diff
@@ -3172,46 +3172,40 @@ Topics:
 #      File: logging-5-6-reference
 #  - Name: Glossary
 #    File: logging-common-terms
-- Name: Distributed tracing
+- Name: Distributed Tracing
   Dir: distr_tracing
   Distros: openshift-enterprise
   Topics:
-  - Name: Release notes
+  - Name: Release notes for the Distributed Tracing Platform
     File: distr-tracing-rn
-  - Name: Distributed tracing architecture
-    Dir: distr_tracing_arch
-    Topics:
-    - Name: Distributed tracing architecture
-      File: distr-tracing-architecture
-  - Name: Distributed tracing platform (Tempo)
-    Dir: distr_tracing_tempo
-    Topics:
-    - Name: Installing
-      File: distr-tracing-tempo-installing
-    - Name: Configuring
-      File: distr-tracing-tempo-configuring
-    - Name: Troubleshooting
-      File: distr-tracing-tempo-troubleshooting
-    - Name: Upgrading
-      File: distr-tracing-tempo-updating
-    - Name: Removing
-      File: distr-tracing-tempo-removing
-  - Name: Distributed tracing platform (Jaeger)
+  - Name: About the Distributed Tracing Platform
+    File: distr-tracing-architecture
+  - Name: Installing the Distributed Tracing Platform
+    File: distr-tracing-tempo-installing
+  - Name: Configuring the Distributed Tracing Platform
+    File: distr-tracing-tempo-configuring
+  - Name: Troubleshooting the Distributed Tracing Platform
+    File: distr-tracing-tempo-troubleshooting
+  - Name: Upgrading the Distributed Tracing Platform
+    File: distr-tracing-tempo-updating
+  - Name: Removing the Distributed Tracing Platform
+    File: distr-tracing-tempo-removing
+  - Name: Distributed Tracing Platform (Jaeger)
     Dir: distr_tracing_jaeger
     Topics:
-    - Name: Installing
+    - Name: Installing the Distributed Tracing Platform (Jaeger)
       File: distr-tracing-jaeger-installing
-    - Name: Configuring
+    - Name: Configuring the Distributed Tracing Platform (Jaeger)
       File: distr-tracing-jaeger-configuring
-    - Name: Upgrading
+    - Name: Upgrading the Distributed Tracing Platform (Jaeger)
       File: distr-tracing-jaeger-updating
-    - Name: Removing
+    - Name: Removing the Distributed Tracing Platform (Jaeger)
       File: distr-tracing-jaeger-removing
 - Name: Red Hat build of OpenTelemetry
   Dir: otel
   Distros: openshift-enterprise
   Topics:
-  - Name: Release notes
+  - Name: Release notes for the {OTELName}
```

Comment:
Attributes don't render in the topic map :disappointed: 
![image](https://github.com/user-attachments/assets/500531f3-b749-4830-8f89-edaa46dda207)

Reply:
:sweat_smile: Ouch, I already fixed a similar issue with the other heading.
On it.

Reply:
understandable to _want_ that to work haha

Reply:
Fixed.

Reply:
:slightly_smiling_face: Please check:
![image](https://github.com/user-attachments/assets/ddcf6704-11b8-4097-8959-0e1abf5243e0)